app/scripts/directives/block.js

```diff
@@ -106,7 +106,11 @@ angular.module('confRegistrationWebApp')
               };
 
               $scope.save = function (choice) {
-                choice.amount = choice.amount.replace(',','');
+                if(choice.amount == null){
```

Comment:
Use the isNull function here: https://lodash.com/docs#isNull  Just so it's consistent with other parts of the app.